modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -472,11 +472,22 @@ use {{invokerPackage}}\ObjectSerializer;
         $multipart = false;
 
         {{#queryParams}}
-
         // query params
         {{#isExplode}}
         if (${{paramName}} !== null) {
+        {{#style}}
+            if('form' === '{{style}}' && is_array(${{paramName}})) {
```

Comment:
I may submit a PR later to add a space after `if` and put line 484 in the same line as line 483, i.e. `} else {`